### PR TITLE
Shorten User model example.

### PR DESCRIPTION
**Pull request by DJankauskas**

### CODE REVIEW

File: docs/docs/getting-started.md [hunk 1]
```diff
@@ -56,16 +56,8 @@ export const User = z
   .response({
     id: z.string().uuid(),
 
-    name: z.string().nullable().optional(),
```
Comment (rattrayalex) [inline docs/docs/getting-started.md:59]: Maybe let's keep name in?

Comment (DJankauskas) [reply docs/docs/getting-started.md:59]: Fair, that's pretty core user data.

File: docs/docs/getting-started.md [hunk 2]
```diff
@@ -56,16 +56,8 @@ export const User = z
...
     username: z.string().nullable().optional(),
-    bio: z.string().nullable().optional(),
     email: z.string().nullable().optional(),
```
[the remark this answers is not in the document]
Comment (DJankauskas) [reply docs/docs/getting-started.md]: Removed! Is the idea that models stored as-is in a db should never have optional fields? 

Also, could this be a pristine convention?

[the remark this answers is not in the document]
Comment (rattrayalex) [reply docs/docs/getting-started.md]: An optional field means the key may not be present in the response. It's common for fields to be nullable, but it's usually clearest to the user to show the key with a null value. 

I love how you're thinking about that – yes, absolutely, it'd be great as a pristine convention! 

We should specify when a field should be optional: for example, only when it's `include`able/`expand`able, or when users may or may not have permissions to see the field.

**Pull request merged**